tests/zeroconfig/windows/testdata/resource_traces/aspnetcore.yaml

```
@@ -3,22 +3,32 @@ resource_spans:
     deployment.environment: zc-iis-test
     host.name: <ANY>
     os.type: windows
+    process.pid: <ANY>
+    process.runtime.description: .NET 6.0.27
+    process.runtime.name: .NET
+    process.runtime.version: 6.0.27
```

Tests can start failing if you upgrade .NET version.
I am not sure if you can add regex here something like
```suggestion
    process.runtime.description: .NET \d.\d.\d+
    process.runtime.name: .NET
    process.runtime.version: \d.\d.\d+
```

Not necessary part of this PR.

Alternatively, you can disable `PROCESSRUNTIME` resource detector.

---

I want to keep the default configuration and this seems a good check to keep in place.

The idea of using regular expressions is very good and the the `testutils` package support that via the `<RE2()>` tag, unfortunately it is not implemented for spans, only for metrics and logs. I will create a KTLO ticket to add it to spans and link to this comment so we can update this test when that is in place.

---

https://splunk.atlassian.net/browse/OTL-2617